META.json

```diff
@@ -6,7 +6,7 @@
    "dynamic_config" : 0,
    "generated_by" : "Dist::Zilla version 5.035, Dist::Milla version v1.0.8, CPAN::Meta::Converter version 2.150001",
    "license" : [
-      "gpl_2"
+      "eupl"
```

Comment:
I would not change this here for two reasons:
1) The EUPL 1.2 is supported by Software::License (EUPL_1_2) so you can put it in the dist.ini (which is present)
2) The license for EUPL is `open_source`.